js/function.js

```diff
@@ -0,0 +1,35 @@
+const isStringLength = (string, length) => string.length <= length;
```

Comment:
Название параметров лучше делать более говорящими чтобы по возможности сразу было понятно за что он отвечает
```suggestion
const isStringLength = (string, maxLength) => string.length <= length;
```